test/common/vmmanage.py

```diff
@@ -97,9 +97,12 @@ def build_and_install(install_image, build_image, args):
     args.setdefault("containers", False)
     args.setdefault("address", None)
     try:
-        skip = "cockpit-ostree"
-        if install_image and "fedora-atomic" in install_image:
-            skip = None
+        skips = [ ]
+        if install_image and "fedora-atomic" not in install_image:
```

Comment:
That happens in another pull request https://github.com/cockpit-project/cockpit/pull/4047